packages/account/src/Components/poi/status/unsupported/card-details/selfie-upload.jsx

```diff
@@ -9,7 +9,26 @@ import Uploader from './uploader.jsx';
 import { setInitialValues, validateFields } from './utils';
 import { ROOT_CLASS, SELFIE_DOCUMENT } from '../constants';
 
-const SelfieUpload = ({ initial_values, goBack, onConfirm, onFileDrop }) => {
+const SelfieUpload = ({ initial_values, is_pa_signup, goBack, onConfirm, onFileDrop, setIsSelfieStepEnabled }) => {
+    const [formik_values, setFormikValues] = React.useState({});
+
+    React.useEffect(() => {
+        if (typeof setIsSelfieStepEnabled === 'function') setIsSelfieStepEnabled(formik_values[SELFIE_DOCUMENT.name]);
```

Comment:
```suggestion
        setIsSelfieStepEnabled?.(formik_values[SELFIE_DOCUMENT.name]);
```
@george-usynin-binary Do we need to check the type? 🤔

Reply:
@farzin-deriv  believe that we can use optional chaining